playbooks/roles/matrix/defaults/main.yaml

```diff
@@ -0,0 +1,11 @@
+---
+state: "present"
+
+matrix_image: "matrixdotorg/synapse:latest"
```

Comment:
will it be safe to use dockerhub instead of quay.io  (due to pull limits) ?

Reply:
most likely we will not face it since we only pull from one place. However anyway changed to get from our mirror and also pinned version to make upgrades easier.